tico/serialize/operators/op_avg_pool2d.py

```diff
@@ -81,32 +144,30 @@ def define_padding_node():
             self.graph.add_operator(pad_operator)
             return padded_input_tensor
 
-        if padding is not None:
+        if not self.has_padding(args):
+            # Don't care count_include_pad
+            result = self.define_avgpool_node(
+                [avgpool_input], [node], "VALID", stride, kernel_size
+            )
+        elif count_include_pad is True:
+            # Add padding before avgpool2d
+            # Circle's avgpool2d does not support count_include_pad=True, so we need to add padding manually
             avgpool_input = define_padding_node()
+            result = self.define_avgpool_node(
+                [avgpool_input], [node], "VALID", stride, kernel_size
```

Comment:
May I ask a question? 🙂
if count_include_pad is True, Why we set "VALID"...?

Reply:
Let me share this psuedo code for better understanding!


```py
if count_include_pad == True:
    (Circle cannot represent count_include_pad=True in AvgPool2D. Therefore we manually add zero padding node.)
    DEFINE zero padding node
    DEFINE avgpool node with no padding (valid)
if count_include_pad == False:
    (Lucky! Circle can represent count_include_pad=False)
    define avgpool node with same/valid padding.
    
    (However, it cannot represent all paddings. So, if the padding is not same or valid, we throw an error.)
    if the paddding is neither same nor valid, throw an error.
```

Reply:
Plus, let me revise the statements.

Reply:
@dayo09 
Ah! I understood it 😀
Thanks for letting me know 👍